types/snowball-stemmers/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for snowball-stemmers 0.6
+// Project: https://github.com/mazko/jssnowball#readme
+// Definitions by: Ryan Volum <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class Stemmer {
```

Comment:
This cannot be class and needs to be interface. When I ran below code:
`
var snowballFactory = require('snowball-stemmers');
console.log(typeof snowballFactory.Stemmer);
`

It prints `undefined`. Noticing that there is no object named Stemmer on the factory.

Reply:
@sheetalkamat I've updated the class to be an interface